test/disabled_sample_buck.sh

```diff
@@ -15,8 +15,8 @@ cp -a "$DIR/../samples/buck/." "$BUILD_DIR"
 # Overwrite config to work with test remote machine.
 setTestRemoteMachineInConfig
 
-# Run mainframer.sh that builds Buck project.
-bash "$REPO_DIR"/mainframer.sh 'buck clean && buck build sample'
+# Run mainframer that builds Buck project.
```

Comment:
Here and everywhere else - docs(samples/buck/README.md for this example) contains different form of script invocation.